plugins/modules/azure_rm_virtualnetwork.py

```diff
@@ -218,7 +222,7 @@ def __init__(self):
             state=dict(type='str', default='present', choices=['present', 'absent']),
             location=dict(type='str'),
             address_prefixes_cidr=dict(type='list', aliases=['address_prefixes']),
```

Comment:
```suggestion
            address_prefixes_cidr=dict(type='list', elements='str', aliases=['address_prefixes']),
```

Reply:
@Fred-sun I've updated the code to reflect this change. Thank you for the comment!